### DISCUSSION TITLE
Backpack and Elytra

### DISCUSSION THREAD

**Discussion by Crystal-Spider** [P3pp3rF1y/SophisticatedBackpacks#509] [Ideas]
Hi, I have both Sophisticated Backpack and Curious Elytra, however there is only a single back slot, which prevents me from wearing both a backpack and the elytra.  
It would be nice if this mod could add an extra back slot. Maybe with a configuration option that lets one decide whether to add this extra slot, defaulting to the current behavior.  
This way it would also be possible to have the curio back slot without other mods installed.

**Comment by TheLexoPlexx** [comment]
Alternate Attempt:

A Chestplate-Upgrade with a single Slot for the Backpack, accepting all Chestplate-Like-Items, rendering both the Chestplate and the Backpack and giving you all the abilities and Armor of the Chestplate and also the Backpack-Functionality.

**Comment by P3pp3rF1y** [comment]
There's already configuration in curios that should allow you to do just that - add one more back slot. And the armor backpack / armor combination is just a big can of worms which would likely require a lot of code / compatibility and continuous maintenance.
